src/main/java/cyclops/stream/ReactiveSeq.java

```diff
@@ -3875,19 +3875,17 @@ default T single(final Predicate<? super T> predicate) {
      */
     default <R> ReactiveSeq<R> retry(final Function<? super T, ? extends R> fn, final int retries, final long delay, final TimeUnit timeUnit) {
```

Comment:
Moved the implementation to `Transformable`, had to call `Transformable.retry()` through `FoldableTraversable.super.retry()` as compiler was complaining if I call directly like (`Transformable.super.retry()`) .

Another thought - is `Transformable` is the right place for `retry`? The retry and Tansformable doesn't really match :). Isn't better we create another interface for `retry`? 
OR handle retry like `recover`? what do you think?